java/ql/src/Telemetry/ExternalApi.qll

```diff
@@ -35,17 +42,17 @@ class ExternalApi extends Callable {
         "#" + this.getName() + paramsString(this)
   }
 
+  private string containerAsJar(Container container) {
```

Comment:
This was already a problem before, but it should not be a member predicate, since `this` is unbound.

Reply:
Right. Will fix.